examples/jenkins-master/jenkins-master-template.json

```diff
@@ -0,0 +1,251 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins-master",
+    "annotations": {
+      "description": "Jenkins service with Kubernetes plugin, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
@bparees hmm, i don't we can get rid of git as we will need to run git clone on slave pod... :/ but I can have a look.